app/helpers/bootstrap_flash_helper.rb

```diff
@@ -0,0 +1,30 @@
+module BootstrapFlashHelper
+  ALERT_TYPES = [:success, :info, :warning, :danger] unless const_defined?(:ALERT_TYPES)
```

Comment:
__[rubocop]__
* Line is too long. [88/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)